.github/workflows/versionBump.yml

```diff
@@ -0,0 +1,50 @@
+name: Update pubspec.yaml
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  pubspec-update: 
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal token.
+          fetch-depth: 0 # otherwise, you will failed to push refs to dest repo.
+          ref: ${{ github.event.pull_request.head.sha }} # checks out the branch being merged into `master` 
+
+      - name: Get Latest pubspec.yaml
+        run: | 
+          git fetch 
+          git checkout -m origin/master pubspec.yaml
+          git add pubspec.yaml
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+          git commit -m "Updating pubspec.yaml" -a || echo "No commit needed. pubspec.yaml is already up-to-date"
+               
+      - name: Install Snapcraft
+        uses: samuelmeuli/action-snapcraft@v1.2.0
```

Comment:
who's this?

Reply:
it's like npm basically managing packages or snaps of software. I'm just using a GitHub action from the marketplace that easily installs snap craft into the ubuntu machine

Reply:
@kmathew-pdftron did you have chance to document this plugin yet?

Reply:
using command line commands instead of action